src/arrayMethodSort.js

```diff
@@ -3,9 +3,30 @@
 /**
  * Implement method Sort
  */
+const compareStrings = (firstString, secondString) => {
+  return String(firstString) > String(secondString);
```

Comment:
hm, take a look what return callback of sort
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/sort